### DISCUSSION TITLE
switch kernel line?

### DISCUSSION THREAD

**Discussion by bobblkabb** [zbm-dev/zfsbootmenu#302] [Questions]
How can I switch to 5.18 kernel line? Installed kernel and headers but I end up in a black screen.

**Comment by zdykstra** [comment]
ZFS 2.1.4 is not compatible with 5.18. Once ZFS 2.1.5 is out, you can use it with 5.18.

**Comment by Dragon-debian** [comment]
ZFS 2.1.5 is avilable on void. now you can use ZFS with 5.18.
